### PR TITLE
Fix Dashboard - Custom container executor - command is required - can't be removed

### PR DESCRIPTION
**Pull request by Pravesh-Sudha**
This PR...

## Changes
Updated the field to not be mandatory
-

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by fivenp** [comment]
@Pravesh-Sudha - Thank you so much for your contribution – it makes totally sense to not enforce a cmd to be run.

I'm curious about your specific setup tho to be honest. What executor are you building which doesn't require a run cmd? Would be great if you could share a little bit :)

**Comment by Pravesh-Sudha** [comment]
@fivenp actually sir, I don't know anything about typescript but I started researching about the issue on ChatGpt about it and I came to know about the specific line which need to change. If it is not right, can you provide the guidance for the same?

**Comment by fivenp** [comment]
@Pravesh-Sudha – oh no – the change is good :) 

I just wanted to know why you needed this change?

File: src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const Command: React.FC = () => {
           form.resetFields();
         }}
       >
-        <Form.Item label="Command" required name="command" rules={[required]}>
+        <Form.Item label="Command" required name="command" >
```
Comment (rangoo94) [inline src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx:75]:
```suggestion
        <Form.Item label="Command" name="command">
```

Comment (Pravesh-Sudha) [reply src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx:75]: Ok I got that, can I make a new PR for this?

Comment (rangoo94) [reply src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx:75]: You don't have to make a new PR - you may either:
* add a new commit to your branch with this change, or
* click "Commit suggestion" - this way GitHub will automatically add a commit with this suggestion :)

**Comment by Pravesh-Sudha** [comment]
Sorry for the inconvenience, but I really like your helping nature @fivenp @rangoo94

File: src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx
Comment (rangoo94) [inline src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx:1]: There is a linter warning now, that the `required` is imported but not used here.

> 'required' is defined but never used

Could you delete it from there too? Unfortunately, it's not possible to make a suggestion to the code that is not near changes, so you will need to do it manually.

Now, when you have committed a change via GitHub, you will need to either:
* Get back to your local code, run `git pull` to download the latest commit, and apply the change
* Use `...` dropdown on the right side of the file name in GitHub, select `Edit file`, and apply updates

| Zrzut ekranu 2023-04-14 o 15 20 48 |
|-|

**Review by Pravesh-Sudha — COMMENTED** [review]
I have remove the import 'required' field

**Review by rangoo94 — APPROVED** [review]
Thank you @Pravesh-Sudha for your first PR and bugfix to the TestKube product! :)

**Comment by Pravesh-Sudha** [comment]
Thanks sir @fivenp @rangoo94 for your guidance

**Pull request merged**